google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/BulkMutation.java

```diff
@@ -48,10 +50,33 @@ public static BulkMutation create(String tableId) {
     return new BulkMutation(tableId);
   }
 
+  /**
+   * Creates a new instance of the bulk mutation builder on an AuthorizedView.
+   *
+   * <p>See {@link com.google.cloud.bigtable.admin.v2.models.AuthorizedView} for more details about
+   * an AuthorizedView.
+   */
+  public static BulkMutation createForAuthorizedView(String tableId, String authorizedViewId) {
+    return new BulkMutation(tableId, authorizedViewId);
+  }
+
+  @InternalApi("For internal use only")
```

Comment:
where is this used? Can it be package private?

Reply:
It's used here: https://github.com/googleapis/java-bigtable/pull/2151/files#diff-0bda21f28dd389fa7bff1f50b603396b0e643b4cd1c98ce8b6a3ec3fce0b307fR1595-R1600 so it cannot be private?

Reply:
The javadoc you referenced is not using this method?

Reply:
Sorry about the broken link. This is related to the other discussion that we had, about converting a `MutateRowsOptions` into a `BulkMutation` first, and then calls the `newMutateRowsBatcher()` on the `BulkMutation` object. And you proposed to call the `newMutateRowsBatcher()` on the `MutateRowsOptions` directly. Let's discuss in the other comment.

If we have decided that we should call `newMutateRowsBatcher()` on the `MutateRowsOptions` directly, then I'll remove this internal function.